patches/server/0913-Improve-logging-and-errors.patch

```diff
@@ -4,23 +4,6 @@ Date: Wed, 14 Dec 2022 15:52:11 -0800
 Subject: [PATCH] Improve logging and errors
 
 
-diff --git a/src/main/java/net/minecraft/server/dedicated/Settings.java b/src/main/java/net/minecraft/server/dedicated/Settings.java
-index f6e423a76d4c9cf639f1d44af80d33cf3072f6b5..135fc81414446f24c3adad71f5199c7898a6c1cd 100644
---- a/src/main/java/net/minecraft/server/dedicated/Settings.java
-+++ b/src/main/java/net/minecraft/server/dedicated/Settings.java
-@@ -49,6 +49,12 @@ public abstract class Settings<T extends Settings<T>> {
-     }
- 
-     public static Properties loadFromFile(Path path) {
-+        // Paper start
-+        if (Files.notExists(path)) {
-+            LOGGER.info("Could not find existing {}. Creating with default values...", path.getFileName());
```

Comment:
Are we fine with dropping the logging change? Upstream's fix doesn't include one.